### DISCUSSION TITLE
SMTP config with docker-compose

### DISCUSSION THREAD

**Discussion by string97bean** [Leantime/leantime#982] [General] [answered]
I apologize if I am missing something, but I don't know how to pass the SMTP config to my install using docker-compose.  I used the following command to install, and everything works great, but I don't know how to get email working.  Any help would be appreciated.

git clone https://github.com/Leantime/docker-leantime.git
cd docker-leantime
docker-compose up -d

**Comment by marcelfolaron** [comment, marked as answer]
Hey, just open up the docker-compose.yml file from the cloned repository
and add the following parameters (right where all the other LEAN parameters
are):

LEAN_EMAIL_RETURN
LEAN_EMAIL_USE_SMTP
LEAN_EMAIL_SMTP_HOSTS
LEAN_EMAIL_SMTP_USERNAME
LEAN_EMAIL_SMTP_PASSWORD
LEAN_EMAIL_SMTP_AUTO_TLS
LEAN_EMAIL_SMTP_SECURE
LEAN_EMAIL_SMTP_PORT

The full list of config options can be found in the config sample file:
https://github.com/Leantime/leantime/blob/master/config/configuration.sample.php
In any case it is good practice to update your docker-compose file and
adjust passwords, hash secrets etc.

**Reply by string97bean** [reply]
I feel silly...I did make config changes to that file when I did the initial install and didn't even notice the other LEAN settings in there.  Worked perfectly....thank you!

**Reply by outnetMT** [reply]
> Hey, just open up the docker-compose.yml file from the cloned repository and add the following parameters (right where all the other LEAN parameters are): LEAN_EMAIL_RETURN LEAN_EMAIL_USE_SMTP LEAN_EMAIL_SMTP_HOSTS LEAN_EMAIL_SMTP_USERNAME LEAN_EMAIL_SMTP_PASSWORD LEAN_EMAIL_SMTP_AUTO_TLS LEAN_EMAIL_SMTP_SECURE LEAN_EMAIL_SMTP_PORT The full list of config options can be found in the config sample file: https://github.com/Leantime/leantime/blob/master/config/configuration.sample.php In any case it is good practice to update your docker-compose file and adjust passwords, hash secrets etc.
> […](#)
> On Tue, Sep 20, 2022 at 12:53 PM string97bean ***@***.***> wrote: I apologize if I am missing something, but I don't know how to pass the SMTP config to my install using docker-compose. I used the following command to install, and everything works great, but I don't know how to get email working. Any help would be appreciated. git clone https://github.com/Leantime/docker-leantime.git cd docker-leantime docker-compose up -d — Reply to this email directly, view it on GitHub <#982>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALG4EFWKADRE5AHMYH5QPMTV7HTYXANCNFSM6AAAAAAQRIPBTQ> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

I'm attempting to configure SMTP mail on our docker utilizing portainer env variables; however, password reset is still not working. 
I'm new to this, so any assistance would be greatly appreciated. thank you very much

![image](https://user-images.githubusercontent.com/72732321/214379793-aacaae6a-a5a5-4961-b6e8-8db39a6bbc41.png)

**Comment by marcelfolaron** [comment]
Glad it worked! Let me know if you have any other questions or issues.

**Comment by jacklocke** [comment]
it's normal the message say "sended" but is KO?

I try with 2 different configuration that both working with a phpmailer for example.

how to log?
